scripts/ci/check_compliance.py

```diff
@@ -524,6 +541,11 @@ def run(self):
             if file.endswith((".dts", ".dtsi", ".overlay"))
         ]
 
+        if not self.is_npx_command_available():
+            self.skip(
+                'dts-linter not installed. To avoid skipping this check, '
```

Comment:
"to avoid skipping" is maybe an akward double negation?


```suggestion
                'dts-linter not installed. To run this check, '
```